autoscheduler/frontend/src/types/Course.ts

```diff
@@ -1,9 +1,14 @@
 export default class Course {
-  id: string;
+  id: string
```

Comment:
Gonna have to remove these extra lines. The linter auto-fixed these lines, but they're quite ugly, so I modified the linting rule that regulated these. Once this branch is rebased with `frontend/master`, it will have the new linting rule, but we'll have to undo the new lines manually during rebase most likely. This is mostly a note to myself, btw.